hathorlib/base_transaction.py

```diff
@@ -393,6 +396,24 @@ def is_nft_creation(self) -> bool:
         """Returns True if it's an NFT creation transaction"""
         return False
 
+    def is_standard(self, max_output_script_size: int = settings.MAX_OUTPUT_SCRIPT_SIZE,
+                    allow_non_standard_script: bool = False) -> bool:
+        """Return True is the transaction is standard
+        """
+        is_nft_creation = self.is_nft_creation
```

Comment:
I think we should remove the `@property` of this method and just call it where it is needed. Using `@property` is misleading that it is an `O(1)` operation when it might not be.

Reply:
Humm the idea was to keep the same standard of `is_block`, `is_transaction` but makes sense, this property is different